src/Components/Web.JS/src/StreamingInterop.ts

```diff
@@ -1,23 +1,23 @@
 import { DotNet } from '@microsoft/dotnet-js-interop';
 
 export async function getNextChunk(data: ArrayBufferView | Blob, position: number, nextChunkSize: number): Promise<Uint8Array> {
-    if (data instanceof Blob) {
-        return await getChunkFromBlob(data, position, nextChunkSize);
-    } else {
-        return getChunkFromArrayBufferView(data, position, nextChunkSize);
-    }
+  if (data instanceof Blob) {
+    return await getChunkFromBlob(data, position, nextChunkSize);
+  } else {
+    return getChunkFromArrayBufferView(data, position, nextChunkSize);
+  }
 }
 
 async function getChunkFromBlob(data: Blob, position: number, nextChunkSize: number): Promise<Uint8Array> {
-    const chunkBlob = data.slice(position, position + nextChunkSize);
-    const arrayBuffer = await chunkBlob.arrayBuffer();
-    const nextChunkData = new Uint8Array(arrayBuffer);
-    return nextChunkData;
+  const chunkBlob = data.slice(position, position + nextChunkSize);
```

Comment:
Does our .editorconfig specify the two whitespace thingamagic for .ts files?

Reply:
I'm not seeing it in the `.editorconfig`. I believe this is inherited from the tsconfig defaults.